docs/diagrams/ScheduleActivityCommandCreation.puml

```diff
@@ -0,0 +1,60 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":PlannerParser" as PlannerParser LOGIC_COLOR
+participant ":ScheduleCommandParser" as ScheduleCommandParser LOGIC_COLOR
+participant "a:ScheduleActivityCommand" as ScheduleActivityCommand LOGIC_COLOR
+end box
+
+[-> LogicManager : execute("schedule activity 2 st/1300 et/1400 d/1")
+activate LogicManager
+
+LogicManager -> PlannerParser : parseCommand("schedule activity 2 st/1300 et/1400 d/1")
+activate PlannerParser
+
+create ScheduleCommandParser
+PlannerParser -> ScheduleCommandParser
+activate ScheduleCommandParser
+
+ScheduleCommandParser --> PlannerParser
+deactivate ScheduleCommandParser
+
+PlannerParser -> ScheduleCommandParser : parse("activity 2 st/1300 et/1400 d/1")
+activate ScheduleCommandParser
+
+ScheduleCommandParser -> ScheduleCommandParser : parseActivity("2 st/1300 et/1400 d/1")
+activate ScheduleCommandParser
+
+create ScheduleActivityCommand
+ScheduleCommandParser -> ScheduleActivityCommand
+activate ScheduleActivityCommand
+
+ScheduleActivityCommand --> ScheduleCommandParser : a
+deactivate ScheduleActivityCommand
+
+ScheduleCommandParser --> ScheduleCommandParser : a
+
+deactivate ScheduleCommandParser
+
+ScheduleCommandParser --> PlannerParser : a
+deactivate ScheduleCommandParser
+
+'Hidden arrow to position the destroy marker below the end of the activation bar.
+ScheduleCommandParser -[hidden]-> PlannerParser
+destroy ScheduleCommandParser
+
+PlannerParser --> LogicManager : a
+deactivate PlannerParser
+
+LogicManager -> ScheduleActivityCommand : execute()
```

Comment:
Would be better to use a reference frame to the next diagram since they're inherently linked